wolfcrypt/src/ecc.c

```diff
@@ -12805,6 +12835,9 @@ int wc_ecc_encrypt_ex(ecc_key* privKey, ecc_key* pubKey, const byte* msg,
             case ecAES_256_CTR:
             {
         #if !defined(NO_AES) && defined(WOLFSSL_AES_COUNTER)
+                byte ctr_iv[16];
+                XMEMCPY(ctr_iv, encIv, 12);
```

Comment:
This causes mixed declaration and code... 

```
wolfcrypt/src/ecc.c:12865:17: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]
12865 |                 Aes aes[1];
      |                 ^~~
cc1: all warnings being treated as errors
```

Reply:
Fixed.